lua/conform/formatters/usort.lua

```diff
@@ -0,0 +1,15 @@
+local util = require("conform.util")
+
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/facebook/usort",
+    description = "Safe, minimal import sorting for Python projects.",
+  },
+  command = "usort",
+  args = { "format", "$FILENAME" },
+  stdin = false,
```

Comment:
Could we use the syntax for [formatting via stdin](https://github.com/facebook/usort/issues/8)? It's preferable when the formatter supports it.

Reply:
Resolved.